### PR TITLE
Add examples and howtos about sensors

### PR DESCRIPTION
**Pull request by potiuk**
The examples and docs were missing for a number of built-in sensors. This documentation and examples do not add much but at least give the user information that there are such sensors available when they look at our documentation.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by potiuk** [comment]
Small but I think nice :)

File: airflow/sensors/time_delta.py [hunk 1]
```diff
@@ -28,6 +28,12 @@ class TimeDeltaSensor(BaseSensorOperator):
     Waits for a timedelta after the run's data interval.
 
     :param delta: time length to wait after the data interval before succeeding.
+
+    .. seealso::
+        For more information on how to use this sensor, take a look at the guide:
```
Comment (XD-DENG) [inline airflow/sensors/time_delta.py:33]: Very minor thing: I see sometimes you wrote _"For more information on how to use this **sensor**"_ and sometimes you wrote _"For more information on how to use this **operator**"_. Does it make sense to you to make it more consistent?

File: docs/apache-airflow/howto/operator/time.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    under the License.
+
+
+
+.. _howto/operator:TimeDeltaSensor:
+
+TimeDeltaSensor
+===============
+
+Use the :class:`~airflow.sensors.time_delta.TimeDeltaSensor` to end sensing after specific time
```
Comment (XD-DENG) [inline docs/apache-airflow/howto/operator/time.rst]:
```suggestion
Use the :class:`~airflow.sensors.time_delta.TimeDeltaSensor` to end sensing after specific time.
```

File: docs/apache-airflow/howto/operator/time.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    :language: python
+    :dedent: 4
+    :start-after: [START example_time_delta_sensor_async]
+    :end-before: [END example_time_delta_sensor_async]
+
+
+
+.. _howto/operator:TimeSensor:
+
+TimeDeltaSensor
```
Comment (XD-DENG) [inline docs/apache-airflow/howto/operator/time.rst]:
```suggestion
TimeSensor
```

Wrong title here?

File: docs/apache-airflow/howto/operator/time.rst [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+.. exampleinclude:: /../../airflow/example_dags/example_sensors.py
+    :language: python
+    :dedent: 4
+    :start-after: [START example_time_sensors]
+    :end-before: [END example_time_sensors]
+
+
+.. _howto/operator:TimeSensorAsync:
+
+TimeDeltaSensorAsync
```
Comment (XD-DENG) [inline docs/apache-airflow/howto/operator/time.rst]:
```suggestion
TimeSensorAsync
```

Also seems wrong title here?

**Review by XD-DENG — CHANGES_REQUESTED** [review]
There seems two wrong titles. Plus two minor points.

File: airflow/sensors/time_delta.py (hunk 1 above)
Comment (potiuk) [reply airflow/sensors/time_delta.py:33]: Yeeah. :). I think I missed it in a few places. This is verbatim copied from our "build-docs" warning when such clause is missing and "operator" is there, so I needed to manually change it. Obvioulsy missed it in a few places

**Comment by potiuk** [comment]
All addressed @XD-DENG

**Review by XD-DENG — APPROVED** [review]

**Pull request merged**